### DISCUSSION TITLE
Dates, Tasks and Journal

### DISCUSSION THREAD

**Discussion by Nick222** [zim-desktop-wiki/zim-desktop-wiki#1726] [Q&A]
Hello!

I organized a journal in Zim's notebook - I put in it various important events that have already(!) passed.

For convenience, I mark each such event with brackets "[..]" - so that the program perceives the event as a task and displays it in the list of tasks (there is a calendar in the display of the Journal, but no list).

But, unfortunately, with this method, nothing appears in the list of tasks in the date column.
When I put the same brackets just in the arrival record in a separate regular note and added the date through the "<" symbol, then the dates in the date column in the list of tasks appeared.

Is it possible to somehow make the Journal, in addition to the calendar, display events as a list (for ease of navigation)?

Or is it possible to make it so that, when creating a case on the Journal page, the task plugin would take its date from the title of the entry in the Journal?

Thanks :)

**Comment by jaap-karssenberg** [comment]
Please see the manual of the task list plugin. The option to integrate with
journal pages already exists.

Op za 6 nov. 2021 13:55 schreef Nick222 ***@***.***>:

**Comment by Nick222** [comment]
This work only for "due days" in option - not for "start days"...
